site/src/includes/anchor.js

```diff
@@ -0,0 +1,59 @@
+function createElement(message) {
+	const el = document.createElement('span');
+	el.innerHTML = message;
+	el.classList.add(
+		'animate-from-right-36',
+		'bg-brand-100',
+		'dark:bg-brand-900',
+		'dark:text-brand-400',
+		'dark:text-gray-300',
+		'fixed',
+		'font-medium',
+		'px-2',
+		'py-1',
+		'rounded',
+		'shadow',
+		'text-brand-600',
+		'text-center',
+		'w-36',
+		'z-10',
+		window.innerWidth > '768' ? 'top-20' : 'top-24',
+	);
+	el.setAttribute('role', 'alert');
+
+	document.body.after(el);
+
+	setTimeout(() => {
```

Comment:
Element have to be removed by end of animation not by timeout. Timeout should be as a fallback.

Use transitionend or animationend for subscription.